bspacks/README.md

```diff
@@ -4,27 +4,29 @@ This makes our installation much friendlier to e.g. Windows. BuckleScript curren
 
 ## Build
 
-This whole process needs to happen with OCaml 4.02.3, so make sure you switch to that version first:
+This whole process needs to happen with OCaml 4.02.3, so make sure you have a switch for this version first:
 
-```sh
-opam switch 4.02.3
+```console
+opam create switch 4.02.3
```

Comment:
this should be `opam switch create 4.02.3`

Reply:
Also, this change is unconvincing to me, because it assumes you have a switch with 4.02.3 where the original command would work. 

If you want, you can add wording for people who haven't created the switch, in which case this command would make sense.